src/components/MdDatepicker/MdDatepicker.vue

```diff
@@ -84,6 +84,11 @@
       toggleDialog () {
         if (!isFirefox || this.mdOverrideNative) {
           this.showDialog = !this.showDialog
+          if (this.showDialog) {
+            this.$emit('md-opened')
+          } else {
+            this.$emit('md-closed')
+          }
```

Comment:
I think `md-opened` and `md-closed` for transition `enter` and `leave` is better.

How do you think?

Reply:
I think it's not needed. In this case, the md-opened and md-closed will fire an intention of close and open.